railties/lib/rails/generators/test_unit/system/templates/system_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class ActionSystemTestCase < ActionSystemTest::Base
+  teardown do
+    take_failed_screenshot
+    Capybara.reset_sessions!
+  end
+end
```

Comment:
I was torn about that. I'm thinking - since AJ is included by default and created by default maybe it should be included by default in the generated file. At first I was thinking that you may not want to use AJ, but if it's auto generated then it should probably be included.